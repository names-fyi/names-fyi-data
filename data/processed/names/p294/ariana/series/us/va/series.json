[{"y":1983,"t":6,"m":0,"f":6,"mr":null,"fr":564,"mp":0,"fp":0.0001949127765325017},{"y":1988,"t":5,"m":0,"f":5,"mr":null,"fr":749,"mp":0,"fp":0.00013938447814451383},{"y":1989,"t":10,"m":0,"f":10,"mr":null,"fr":483,"mp":0,"fp":0.0002695127210004312},{"y":1990,"t":18,"m":0,"f":18,"mr":null,"fr":320,"mp":0,"fp":0.00047648039812584375},{"y":1991,"t":23,"m":0,"f":23,"mr":null,"fr":255,"mp":0,"fp":0.00062345829606137},{"y":1992,"t":25,"m":0,"f":25,"mr":null,"fr":237,"mp":0,"fp":0.0006850253459377997},{"y":1993,"t":18,"m":0,"f":18,"mr":null,"fr":304,"mp":0,"fp":0.0005107542137222632},{"y":1994,"t":28,"m":0,"f":28,"mr":null,"fr":226,"mp":0,"fp":0.0007940334061197289},{"y":1995,"t":23,"m":0,"f":23,"mr":null,"fr":253,"mp":0,"fp":0.0006746648675603532},{"y":1996,"t":28,"m":0,"f":28,"mr":null,"fr":230,"mp":0,"fp":0.0008202003632315894},{"y":1997,"t":38,"m":0,"f":38,"mr":null,"fr":186,"mp":0,"fp":0.0011425478817763612},{"y":1998,"t":50,"m":0,"f":50,"mr":null,"fr":151,"mp":0,"fp":0.001474839242522565},{"y":1999,"t":59,"m":0,"f":59,"mr":null,"fr":124,"mp":0,"fp":0.0017175628074874094},{"y":2000,"t":45,"m":0,"f":45,"mr":null,"fr":166,"mp":0,"fp":0.001266677926026009},{"y":2001,"t":70,"m":0,"f":70,"mr":null,"fr":102,"mp":0,"fp":0.0019943019943019944},{"y":2002,"t":59,"m":0,"f":59,"mr":null,"fr":129,"mp":0,"fp":0.0016547004711689477},{"y":2003,"t":82,"m":0,"f":82,"mr":null,"fr":92,"mp":0,"fp":0.0022755654225058972},{"y":2004,"t":79,"m":0,"f":79,"mr":null,"fr":101,"mp":0,"fp":0.0021361743550916664},{"y":2005,"t":96,"m":0,"f":96,"mr":null,"fr":82,"mp":0,"fp":0.002587391855106056},{"y":2006,"t":101,"m":0,"f":101,"mr":null,"fr":79,"mp":0,"fp":0.002667793655405584},{"y":2007,"t":98,"m":0,"f":98,"mr":null,"fr":82,"mp":0,"fp":0.0025479031796791724},{"y":2008,"t":123,"m":0,"f":123,"mr":null,"fr":60,"mp":0,"fp":0.0032705807275047864},{"y":2009,"t":80,"m":0,"f":80,"mr":null,"fr":94,"mp":0,"fp":0.0021806089350451113},{"y":2010,"t":95,"m":0,"f":95,"mr":null,"fr":80,"mp":0,"fp":0.0026653947589922},{"y":2011,"t":85,"m":0,"f":85,"mr":null,"fr":90,"mp":0,"fp":0.0023580325685910062},{"y":2012,"t":90,"m":0,"f":90,"mr":null,"fr":87,"mp":0,"fp":0.0024746349913387774},{"y":2013,"t":111,"m":0,"f":111,"mr":null,"fr":70,"mp":0,"fp":0.003060802426582104},{"y":2014,"t":166,"m":0,"f":166,"mr":null,"fr":33,"mp":0,"fp":0.004449686377526403},{"y":2015,"t":125,"m":0,"f":125,"mr":null,"fr":55,"mp":0,"fp":0.0033373381391002535},{"y":2016,"t":136,"m":0,"f":136,"mr":null,"fr":48,"mp":0,"fp":0.003663891807430157},{"y":2017,"t":107,"m":0,"f":107,"mr":null,"fr":70,"mp":0,"fp":0.002972057107938448},{"y":2018,"t":95,"m":0,"f":95,"mr":null,"fr":74,"mp":0,"fp":0.0026454289771936177},{"y":2019,"t":97,"m":0,"f":97,"mr":null,"fr":72,"mp":0,"fp":0.002753022648578078},{"y":2020,"t":76,"m":0,"f":76,"mr":null,"fr":94,"mp":0,"fp":0.0022310289153089683},{"y":2021,"t":78,"m":0,"f":78,"mr":null,"fr":92,"mp":0,"fp":0.0022611317254174397},{"y":2022,"t":71,"m":0,"f":71,"mr":null,"fr":101,"mp":0,"fp":0.002054398148148148}]