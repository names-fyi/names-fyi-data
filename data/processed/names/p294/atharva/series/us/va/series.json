[{"y":2020,"t":5,"m":5,"f":0,"mr":1026,"fr":null,"mp":0.0001295638879531497,"fp":0}]