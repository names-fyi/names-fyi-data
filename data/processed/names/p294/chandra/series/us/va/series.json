[{"y":1955,"t":5,"m":0,"f":5,"mr":null,"fr":575,"mp":0,"fp":0.00012212099748430745},{"y":1956,"t":6,"m":0,"f":6,"mr":null,"fr":522,"mp":0,"fp":0.0001451484142535743},{"y":1957,"t":8,"m":0,"f":8,"mr":null,"fr":472,"mp":0,"fp":0.000193026903124623},{"y":1958,"t":8,"m":0,"f":8,"mr":null,"fr":467,"mp":0,"fp":0.00019591516873193908},{"y":1959,"t":7,"m":0,"f":7,"mr":null,"fr":515,"mp":0,"fp":0.0001724945171385624},{"y":1960,"t":5,"m":0,"f":5,"mr":null,"fr":610,"mp":0,"fp":0.00012557450335283924},{"y":1963,"t":6,"m":0,"f":6,"mr":null,"fr":551,"mp":0,"fp":0.00015080302611405737},{"y":1964,"t":10,"m":0,"f":10,"mr":null,"fr":436,"mp":0,"fp":0.0002509599217005044},{"y":1965,"t":9,"m":0,"f":9,"mr":null,"fr":447,"mp":0,"fp":0.00024671728939938047},{"y":1966,"t":8,"m":0,"f":8,"mr":null,"fr":457,"mp":0,"fp":0.00023340627279358134},{"y":1967,"t":7,"m":0,"f":7,"mr":null,"fr":488,"mp":0,"fp":0.00020908004778972522},{"y":1968,"t":11,"m":0,"f":11,"mr":null,"fr":373,"mp":0,"fp":0.0003337581163905577},{"y":1969,"t":11,"m":0,"f":11,"mr":null,"fr":367,"mp":0,"fp":0.0003295682655721005},{"y":1971,"t":16,"m":0,"f":16,"mr":null,"fr":303,"mp":0,"fp":0.0004839831815844399},{"y":1972,"t":11,"m":0,"f":11,"mr":null,"fr":362,"mp":0,"fp":0.0003714710252600297},{"y":1973,"t":16,"m":0,"f":16,"mr":null,"fr":283,"mp":0,"fp":0.0005697194131890044},{"y":1974,"t":13,"m":0,"f":13,"mr":null,"fr":328,"mp":0,"fp":0.00046821537907437423},{"y":1975,"t":15,"m":0,"f":15,"mr":null,"fr":303,"mp":0,"fp":0.0005541803672368567},{"y":1976,"t":14,"m":0,"f":14,"mr":null,"fr":304,"mp":0,"fp":0.0005204267499349467},{"y":1977,"t":11,"m":0,"f":11,"mr":null,"fr":365,"mp":0,"fp":0.00039014009576165986},{"y":1978,"t":7,"m":0,"f":7,"mr":null,"fr":497,"mp":0,"fp":0.000251364550416547},{"y":1979,"t":9,"m":0,"f":9,"mr":null,"fr":441,"mp":0,"fp":0.00030832476875642344},{"y":1980,"t":11,"m":0,"f":11,"mr":null,"fr":380,"mp":0,"fp":0.0003689294338610142},{"y":1981,"t":10,"m":0,"f":10,"mr":null,"fr":406,"mp":0,"fp":0.0003326790645064706},{"y":1982,"t":6,"m":0,"f":6,"mr":null,"fr":573,"mp":0,"fp":0.0001948178453146308},{"y":1984,"t":5,"m":0,"f":5,"mr":null,"fr":656,"mp":0,"fp":0.00015937778911130946},{"y":1985,"t":10,"m":0,"f":10,"mr":null,"fr":425,"mp":0,"fp":0.0003034809262237868},{"y":1986,"t":5,"m":0,"f":5,"mr":null,"fr":713,"mp":0,"fp":0.00014821402104639098},{"y":1988,"t":7,"m":0,"f":7,"mr":null,"fr":590,"mp":0,"fp":0.00019513826940231936},{"y":1989,"t":8,"m":0,"f":8,"mr":null,"fr":575,"mp":0,"fp":0.00021561017680034498},{"y":1990,"t":8,"m":0,"f":8,"mr":null,"fr":574,"mp":0,"fp":0.00021176906583370833},{"y":1991,"t":6,"m":0,"f":6,"mr":null,"fr":714,"mp":0,"fp":0.0001626412946247052},{"y":1993,"t":5,"m":0,"f":5,"mr":null,"fr":820,"mp":0,"fp":0.00014187617047840644},{"y":1996,"t":5,"m":0,"f":5,"mr":null,"fr":853,"mp":0,"fp":0.00014646435057706953}]