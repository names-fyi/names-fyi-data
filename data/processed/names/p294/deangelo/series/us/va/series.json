[{"y":1963,"t":5,"m":5,"f":0,"mr":493,"fr":null,"mp":0.00011586411456643648,"fp":0},{"y":1977,"t":5,"m":5,"f":0,"mr":490,"fr":null,"mp":0.00015348722986247544,"fp":0},{"y":1978,"t":6,"m":6,"f":0,"mr":438,"fr":null,"mp":0.00018727760784068918,"fp":0},{"y":1980,"t":5,"m":5,"f":0,"mr":501,"fr":null,"mp":0.00014753179310141335,"fp":0},{"y":1981,"t":6,"m":6,"f":0,"mr":446,"fr":null,"mp":0.00017348561515107705,"fp":0},{"y":1983,"t":7,"m":7,"f":0,"mr":408,"fr":null,"mp":0.00020008575103615836,"fp":0},{"y":1984,"t":5,"m":5,"f":0,"mr":511,"fr":null,"mp":0.00013912849908175192,"fp":0},{"y":1985,"t":7,"m":7,"f":0,"mr":431,"fr":null,"mp":0.00018518028623581387,"fp":0},{"y":1986,"t":8,"m":8,"f":0,"mr":410,"fr":null,"mp":0.00021152269902963963,"fp":0},{"y":1987,"t":14,"m":14,"f":0,"mr":306,"fr":null,"mp":0.00035514066106897337,"fp":0},{"y":1988,"t":12,"m":12,"f":0,"mr":348,"fr":null,"mp":0.00029908032799142635,"fp":0},{"y":1989,"t":9,"m":9,"f":0,"mr":436,"fr":null,"mp":0.00021496130696474635,"fp":0},{"y":1990,"t":14,"m":14,"f":0,"mr":353,"fr":null,"mp":0.0003230197734247018,"fp":0},{"y":1991,"t":7,"m":7,"f":0,"mr":553,"fr":null,"mp":0.0001663774867492216,"fp":0},{"y":1992,"t":7,"m":7,"f":0,"mr":559,"fr":null,"mp":0.00016648036720812425,"fp":0},{"y":1993,"t":11,"m":11,"f":0,"mr":420,"fr":null,"mp":0.0002678158400895966,"fp":0},{"y":1995,"t":9,"m":9,"f":0,"mr":509,"fr":null,"mp":0.00022578460149017836,"fp":0},{"y":1996,"t":10,"m":10,"f":0,"mr":463,"fr":null,"mp":0.00025636424231548184,"fp":0},{"y":1997,"t":12,"m":12,"f":0,"mr":416,"fr":null,"mp":0.000310896937665164,"fp":0},{"y":1998,"t":5,"m":5,"f":0,"mr":773,"fr":null,"mp":0.00012601759205585098,"fp":0},{"y":1999,"t":5,"m":5,"f":0,"mr":759,"fr":null,"mp":0.00012550830865003262,"fp":0},{"y":2000,"t":6,"m":6,"f":0,"mr":706,"fr":null,"mp":0.00014622733476311173,"fp":0},{"y":2001,"t":8,"m":8,"f":0,"mr":593,"fr":null,"mp":0.00019618421697974398,"fp":0},{"y":2002,"t":5,"m":5,"f":0,"mr":803,"fr":null,"mp":0.0001235086332534644,"fp":0},{"y":2003,"t":5,"m":5,"f":0,"mr":847,"fr":null,"mp":0.00012061853183123055,"fp":0},{"y":2004,"t":5,"m":5,"f":0,"mr":901,"fr":null,"mp":0.00011718933108329818,"fp":0},{"y":2005,"t":6,"m":6,"f":0,"mr":811,"fr":null,"mp":0.00013933398355858994,"fp":0},{"y":2006,"t":7,"m":7,"f":0,"mr":753,"fr":null,"mp":0.00015926827603467497,"fp":0},{"y":2007,"t":8,"m":8,"f":0,"mr":706,"fr":null,"mp":0.00018058283108733438,"fp":0},{"y":2010,"t":6,"m":6,"f":0,"mr":862,"fr":null,"mp":0.00014441823520916576,"fp":0},{"y":2011,"t":9,"m":9,"f":0,"mr":626,"fr":null,"mp":0.0002179968511565944,"fp":0},{"y":2012,"t":10,"m":10,"f":0,"mr":601,"fr":null,"mp":0.00023920011481605511,"fp":0},{"y":2017,"t":6,"m":6,"f":0,"mr":895,"fr":null,"mp":0.00014740203906154036,"fp":0},{"y":2018,"t":7,"m":7,"f":0,"mr":817,"fr":null,"mp":0.00017199439789675422,"fp":0},{"y":2020,"t":7,"m":7,"f":0,"mr":840,"fr":null,"mp":0.00018138944313440957,"fp":0},{"y":2022,"t":5,"m":5,"f":0,"mr":1107,"fr":null,"mp":0.00013135426244581636,"fp":0}]