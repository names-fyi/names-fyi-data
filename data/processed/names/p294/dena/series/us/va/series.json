[{"y":1953,"t":9,"m":0,"f":9,"mr":null,"fr":401,"mp":0,"fp":0.0002276521475185916},{"y":1954,"t":8,"m":0,"f":8,"mr":null,"fr":434,"mp":0,"fp":0.00019634310958399804},{"y":1955,"t":9,"m":0,"f":9,"mr":null,"fr":417,"mp":0,"fp":0.0002198177954717534},{"y":1957,"t":10,"m":0,"f":10,"mr":null,"fr":418,"mp":0,"fp":0.00024128362890577873},{"y":1959,"t":9,"m":0,"f":9,"mr":null,"fr":456,"mp":0,"fp":0.00022177866489243733},{"y":1960,"t":7,"m":0,"f":7,"mr":null,"fr":500,"mp":0,"fp":0.00017580430469397494},{"y":1961,"t":13,"m":0,"f":13,"mr":null,"fr":373,"mp":0,"fp":0.0003237132398715107},{"y":1962,"t":14,"m":0,"f":14,"mr":null,"fr":352,"mp":0,"fp":0.0003479730569433052},{"y":1963,"t":17,"m":0,"f":17,"mr":null,"fr":321,"mp":0,"fp":0.00042727524065649583},{"y":1964,"t":11,"m":0,"f":11,"mr":null,"fr":412,"mp":0,"fp":0.0002760559138705549},{"y":1965,"t":11,"m":0,"f":11,"mr":null,"fr":393,"mp":0,"fp":0.0003015433537103539},{"y":1966,"t":18,"m":0,"f":18,"mr":null,"fr":284,"mp":0,"fp":0.000525164113785558},{"y":1967,"t":17,"m":0,"f":17,"mr":null,"fr":297,"mp":0,"fp":0.0005077658303464755},{"y":1968,"t":22,"m":0,"f":22,"mr":null,"fr":241,"mp":0,"fp":0.0006675162327811154},{"y":1969,"t":22,"m":0,"f":22,"mr":null,"fr":267,"mp":0,"fp":0.000659136531144201},{"y":1970,"t":21,"m":0,"f":21,"mr":null,"fr":274,"mp":0,"fp":0.0006103586583735395},{"y":1971,"t":14,"m":0,"f":14,"mr":null,"fr":321,"mp":0,"fp":0.00042348528388638497},{"y":1972,"t":14,"m":0,"f":14,"mr":null,"fr":313,"mp":0,"fp":0.00047278130487640144},{"y":1973,"t":14,"m":0,"f":14,"mr":null,"fr":318,"mp":0,"fp":0.0004985044865403788},{"y":1974,"t":10,"m":0,"f":10,"mr":null,"fr":383,"mp":0,"fp":0.0003601656762110571},{"y":1975,"t":9,"m":0,"f":9,"mr":null,"fr":410,"mp":0,"fp":0.000332508220342114},{"y":1976,"t":6,"m":0,"f":6,"mr":null,"fr":543,"mp":0,"fp":0.0002230400356864057},{"y":1977,"t":14,"m":0,"f":14,"mr":null,"fr":320,"mp":0,"fp":0.0004965419400602943},{"y":1978,"t":6,"m":0,"f":6,"mr":null,"fr":555,"mp":0,"fp":0.00021545532892846883},{"y":1979,"t":11,"m":0,"f":11,"mr":null,"fr":389,"mp":0,"fp":0.00037684138403562865},{"y":1980,"t":6,"m":0,"f":6,"mr":null,"fr":589,"mp":0,"fp":0.0002012342366514623},{"y":1981,"t":8,"m":0,"f":8,"mr":null,"fr":486,"mp":0,"fp":0.00026614325160517646},{"y":1982,"t":5,"m":0,"f":5,"mr":null,"fr":666,"mp":0,"fp":0.00016234820442885902},{"y":1983,"t":5,"m":0,"f":5,"mr":null,"fr":660,"mp":0,"fp":0.00016242731377708475},{"y":1984,"t":8,"m":0,"f":8,"mr":null,"fr":483,"mp":0,"fp":0.0002550044625780951},{"y":1986,"t":6,"m":0,"f":6,"mr":null,"fr":623,"mp":0,"fp":0.0001778568252556692},{"y":1988,"t":9,"m":0,"f":9,"mr":null,"fr":505,"mp":0,"fp":0.0002508920606601249},{"y":1989,"t":5,"m":0,"f":5,"mr":null,"fr":799,"mp":0,"fp":0.0001347563605002156}]