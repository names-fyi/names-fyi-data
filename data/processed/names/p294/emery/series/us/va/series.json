[{"y":1915,"t":7,"m":7,"f":0,"mr":293,"fr":null,"mp":0.0003878975950349108,"fp":0},{"y":1916,"t":7,"m":7,"f":0,"mr":296,"fr":null,"mp":0.0003794037940379404,"fp":0},{"y":1919,"t":9,"m":9,"f":0,"mr":269,"fr":null,"mp":0.0004131093362709997,"fp":0},{"y":1921,"t":6,"m":6,"f":0,"mr":371,"fr":null,"mp":0.0002438132390588809,"fp":0},{"y":1925,"t":5,"m":5,"f":0,"mr":410,"fr":null,"mp":0.0002145646483285414,"fp":0},{"y":1927,"t":6,"m":6,"f":0,"mr":360,"fr":null,"mp":0.00025960539979231567,"fp":0},{"y":1928,"t":5,"m":5,"f":0,"mr":408,"fr":null,"mp":0.00022123893805309734,"fp":0},{"y":1934,"t":8,"m":8,"f":0,"mr":288,"fr":null,"mp":0.00036598197538771215,"fp":0},{"y":2005,"t":12,"m":5,"f":7,"mr":938,"fr":886,"mp":0.00011611165296549161,"fp":0.0001886639894348166},{"y":2007,"t":14,"m":6,"f":8,"mr":873,"fr":845,"mp":0.00013543712331550077,"fp":0.00020799209630034058},{"y":2008,"t":20,"m":8,"f":12,"mr":720,"fr":615,"mp":0.0001862370798025887,"fp":0.0003190810465858328},{"y":2009,"t":24,"m":8,"f":16,"mr":704,"fr":485,"mp":0.00018980734554427256,"fp":0.0004361217870090223},{"y":2010,"t":23,"m":6,"f":17,"mr":871,"fr":443,"mp":0.00014441823520916576,"fp":0.00047696537792492},{"y":2011,"t":29,"m":5,"f":24,"mr":978,"fr":334,"mp":0.00012110936175366356,"fp":0.0006657974311315782},{"y":2012,"t":41,"m":14,"f":27,"mr":469,"fr":309,"mp":0.0003348801607424772,"fp":0.0007423904974016332},{"y":2013,"t":47,"m":12,"f":35,"mr":530,"fr":229,"mp":0.00028772838440512156,"fp":0.0009651178822556183},{"y":2014,"t":59,"m":5,"f":54,"mr":1042,"fr":156,"mp":0.0001182116935007211,"fp":0.0014474883396772638},{"y":2015,"t":63,"m":6,"f":57,"mr":921,"fr":151,"mp":0.00014150943396226416,"fp":0.0015218261914297156},{"y":2016,"t":68,"m":17,"f":51,"mr":447,"fr":160,"mp":0.00040874227597316725,"fp":0.0013739594277863089},{"y":2017,"t":61,"m":7,"f":54,"mr":814,"fr":144,"mp":0.00017196904557179707,"fp":0.0014999166712960392},{"y":2018,"t":75,"m":12,"f":63,"mr":560,"fr":125,"mp":0.00029484753925157866,"fp":0.001754337111191557},{"y":2019,"t":61,"m":13,"f":48,"mr":541,"fr":169,"mp":0.00032962296204264814,"fp":0.0013623204858943065},{"y":2020,"t":62,"m":6,"f":56,"mr":953,"fr":128,"mp":0.00015547666554377963,"fp":0.0016439160428592396},{"y":2021,"t":77,"m":9,"f":68,"mr":730,"fr":107,"mp":0.00023077514808072,"fp":0.001971243042671614},{"y":2022,"t":77,"m":13,"f":64,"mr":556,"fr":116,"mp":0.00034152108235912257,"fp":0.001851851851851852}]