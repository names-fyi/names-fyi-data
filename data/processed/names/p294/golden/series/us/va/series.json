[{"y":1918,"t":5,"m":0,"f":5,"mr":null,"fr":439,"mp":0,"fp":0.000184243496204584},{"y":1921,"t":12,"m":5,"f":7,"mr":427,"fr":377,"mp":0.00020317769921573408,"fp":0.00023557126030624264},{"y":1926,"t":6,"m":6,"f":0,"mr":353,"fr":null,"mp":0.0002708803611738149,"fp":0}]